lsp/core/aws-lsp-core/src/index.ts

```diff
@@ -1,5 +1,6 @@
 // todo : IDE-10874 : move core support files into this package from pre-monorepo locations
 export * from './content/schemaProvider'
```

Comment:
nit: we should stick with camelCase folder names to keep things consistent

Reply:
I've taken a note to revisit this as we wrap up explorations and start formalizing things in the lsp repo